doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1688 @@
+=pod
+
+=head1 NAME
+
+migration_guide - OpenSSL migration guide
+
+=head1 SYNOPSIS
+
+See the individual manual pages for details.
+
+=head1 DESCRIPTION
+
+In OpenSSL 3.0 the concept of a B<OSSL_LIB_CTX> library context has been added.
+A default NULL library context is used if no library context is created. This
+allows applications to work as they did in 1.1.1 with minimal changes.
+If the user creates an  B<OSSL_LIB_CTX> via OSSL_LIB_CTX_new() then many
+functions need to be changed to pass additional parameters to handle the
+library context.
+
+=head2 Functions that should be changed if B<OSSL_LIB_CTX> is used
+
+All functions listed below with a I<NAME> have a replacment function I<NAME_ex>
+that takes B<OSSL_LIB_CTX> as an additional argument. Functions that have other
+mappings are listed.
```

Comment:
listed -> listed along with the respective name

Reply:
@slontis, I wonder why this and other (minor) comments are marked as resolved without any changes?

Reply:
I have not committed the changes yet - but I have processed them locally..

Reply:
I see. To avoid such confusion, I push first before giving reactions to review comments.

Reply:
I like to do them as I go, so I can keep track of what I have done (especially if there are going to be lots of them)

Reply:
Yeah, I also often have this problem.
What i then tend to do is to prepare responses in the browser an push the "Comment" buttons after updating the PR.
Would be nice if GitHub offered a way of delaying responses and then sending them out in a batch, like there is for review comments.